ios/FluentUI/Popup Menu/PopupMenuController.swift

```diff
@@ -98,6 +101,17 @@ open class PopupMenuController: DrawerController {
             }
         }
     }
+
+    @objc open func searchBar(isVisible: Bool, placeholderText: String? = "") {
+        searchBar.isHidden = !isVisible
+        searchBar.placeholderText = placeholderText
+        adjustsHeightForKeyboard = true
+    }
+
+    @objc open var searchText: String? {
```

Comment:
Will this be overridden at this level or should it be a readonly property?

Reply:
I think this should be a read only, but I couldn't think of any scenarios where we'd want to overwrite the search text content